index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    Hello World, we are software developers!
```

Comment:
Use semantic tags eg main or a section.😃

Reply:
- [ ] Kindly, make sure that Your suggested changes phrased the comments politely, as suggestions instead of orders and always includes an explanation, a "why", for each requested change.
For Example : "I think it would be better to use semantic tags because it would improve the HTML structure and make the page more accessible "